ansible-advancedlinux/azuredeploy.json

```diff
@@ -1,9 +1,11 @@
 {
+
   "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
     "location": {
       "type": "string",
+      "defaultValue": "West US",
```

Comment:
Please remove the default value and let the user choose it. Also there are more locations for a VM deployment now. Might be best to remove the allowed locations list